packages/nocodb/src/lib/db/sql-client/lib/sqlite/SqliteClient.ts

```diff
@@ -7,6 +7,9 @@ import Result from '../../../util/Result';
 import queries from './sqlite.queries';
 import lodash from 'lodash';
 import _ from 'lodash';
```

Comment:
These 2 lines basically load the entire lodash library. Yet I don't know why we import it twice.

Since we just need `isEmpty`, `mapKeys`, and `find` in this file. We may just import as below.

```js
import isEmpty from 'lodash/isEmpty';
import mapKeys from 'lodash/mapKeys';
import find from 'lodash/find';
```

In this way, we enable tree-shaking in lodash so that we could have a smaller bundle. You may verify it and change other DB clients.

Reply:
Good catch, fixed this in all other clients as well, please take a look and resolve if it is good